src/frontend/apps/hub/src/features/chat/components/ChatComposer.scss

```
@@ -148,11 +148,18 @@
   }
 
   &__input {
+    display: block;
+    box-sizing: border-box;
     width: 100%;
+    // Eight 22px text rows plus the 2px vertical padding on each side.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)